packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -1,12 +1,14 @@
 pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
 
+import "@openzeppelin/contracts/utils/SafeCast.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 
 import "../../common/implementation/Lockable.sol";
 import "../../common/implementation/FixedPoint.sol";
 import "../../common/implementation/Timer.sol";
```

Comment:
this import is not used as you've made it Testable.

Reply:
+1

Reply:
Done.